src/components/FileItem/components/CompressedVariant/CompressedVariant.props.ts

```diff
@@ -0,0 +1,16 @@
+import { MouseEvent } from 'react';
+
+import { CompressedVariantConfig } from './CompressedVariant.styles';
+import { FileItemState } from '../../types';
+
+export type CompressedVariantProps = {
+  custom?: CompressedVariantConfig;
+  state: FileItemState;
+  file: File;
+  isInverted?: boolean;
+  uploadedPercentage?: number;
+  alertText?: string;
+  onReplaceClick?: (e?: MouseEvent) => void;
```

Comment:
Same as above. Maybe we could import common properties? It's just an inner component of `FileItem` 🤔

Reply:
I extracted common properties called: `FileItemCommonProps`. Unfortunately I had to locate it inside types.ts file instead of `FileItem.props.ts` because of circular dependencies.